frontend/packages/client/src/components/Community/CommunityEditorDetails.js

```diff
@@ -80,6 +89,11 @@ export const CommunityUsersForm = ({
                     width: '100%',
                     ...(isInvalid ? {} : undefined),
                   }}
+                  ref={
+                    addrList.length === 1 && addr === '' && autoFocusOnLoad
+                      ? refOnFirstInput
+                      : undefined
+                  }
```

Comment:
this component is used twice in the same step so the last focus will win. For that reason using autoFocusOnLoad to enable one and disable the second one. That way the first input will keep the focus.